docs/quickstart/assessment/standalone-assessment.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2021 Learnosity, MIT License
```

Comment:
I assume this file was added accidentally?

Reply:
Yes. Sorry I have this approach of putting placeholder things in here, to remind me to put something in that does what it did. I'll remove. :-)

Reply:
Removed here: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/0c5500ec37c959b273105a3f13853be0231cca1b